clients/java/build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'io.odpf'
-version '1.0.0-SNAPSHOT'
+version '1.0.0-rc-1'
```

Comment:
@AkbaraliShaikh We should keep the version of the java client to same as the core raccoon version. It will be tricky to manage two different ones.

Reply:
This `rc` version is to test the `JAR`, once all looks good than I will change it to accordingly.

Reply:
raccoon version is at `0.1.5` So even for testing we should keep java client version at `0.1.5-rc` instead of `1.0.0`

Reply:
Updated.